### DISCUSSION TITLE
Newbie havng problems with NavBar links disapearing when screen <768

### DISCUSSION THREAD

**Discussion by RaycasterV** [twbs/bootstrap#37950] [Q&A]
This is a simple navbar NavBrand - navlink1 "home" - navlink2 "Page 2".
I had my navbar working great and the the 2 navlinks dropped side by side under the NavBrand when the screen was squeezed to mobile size. I then gave the background to each navlink a different color and rounded the corners like a button. (Don't want a button....)
But now the navlinks disappear instead of under the Navbrand when <768

HTML:

    <nav class="navbar navbar-expand-sm my-custom-navbar mb-4">
      <div class="container-fluid">
        <div class="navbar-header">
            <span class="navbar-brand">Fishing Lake Ontario</span>
        </div>
        <div class="collapse navbar-collapse" id="navbarNav">
          <ul class="navbar-nav ms-auto">
            <li class="nav-item mx-2">
              <a class="nav-link custom-link" href="index.html">Home</a>
            </li>
            <li class="nav-item">
              <a class="nav-link custom-link disabled" href="page2.html">Page 2</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>
    

CSS:

.my-custom-navbar {
  background-color: rgba(44,130,201, 0.2);
  font-family: 'Consolas', monospace;
  font-size: 1.2rem;
  font-weight: 800;
}
h5{
  color: red;
  font-family: Arial, sans-serif;
  font-size: 1.5rem;
}

.navbar-brand
{
   text-transform: uppercase;
  background-image: linear-gradient(
    -225deg,
    darkblue 0%,
    mediumblue 29%,
    lightblue 67%,
    white 100%
  );
  background-size: auto auto;
  background-clip: border-box;
  background-size: 200% auto;
  color: #fff;
  background-clip: text;
  text-fill-color: transparent;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  animation: textclip 20s linear infinite;
  display: inline-block;
      font-size: 25px;
}

@keyframes textclip {
  to {
    background-position: 200% center;
  }
}
.card{
  padding: 1.5em .5em .5em;
  border-radius: .5em;
  text-align: center;
  box-shadow: 0 5px 10px rgba(0,0,0,.2);
}
.card-body {
  text-align: left;
}
.custom-link {
  display: block!important; 
  background-color: #158fe4; /* dark blue */
  color: white; /* make text color white */
  padding: 0.25rem 0.5rem; /* add padding for readability */
  border-radius: 0.25rem; /* add rounded corners */
}
@media only screen and (max-width: 768px) {
  .navbar{
    flex-wrap: nowrap;
  }
}

Any help very much appreciated. I think I'm on my 6th youtube going down a rabbit hole...

**Comment by ExtravagantBoost** [comment]
because you have the `navbar-expand` class within your navbar, this turns your navbar into a collapsible navbar meaning when the screen is too small to fit many buttons on it, which replaces the buttons into a single button used to open/close the navbar, because you do not have this button coded in, the button used will  not appear when you need it to show the other buttons
